packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1193,6 +1193,35 @@ oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Email
 oie.user.profile.secondary.email= Secondary email
+oie.user.profile.middleName = Middle name
+oie.user.profile.honorificPrefix = Honorific prefix
+oie.user.profile.honorificSuffix = Honorific suffix
+oie.user.profile.primaryEmail = Primary email
+oie.user.profile.title = Title
+oie.user.profile.displayName = Display name
+oie.user.profile.nickName = Nickname
+oie.user.profile.profileUrl = Profile URL
+oie.user.profile.secondEmail = Secondary email
```

Comment:
This value is already present with `oie.user.profile.secondary.email`. Can we leverage this existing key?

Reply:
Good catch! I'll fix that